deployments/fpga_plugin/intel-fpga-initcontainer.Dockerfile

```diff
@@ -0,0 +1,79 @@
+### 1. Temporary image to prepare AOCL and OPAE components
+FROM centos:centos7.4.1708 as temporary
+
+# install aocl and opae deps
+RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+RUN yum -y install which perl uuid libuuid-devel json-c
+
+ADD a10_gx_pac_ias_1_1_pv_rte_installer.tar.gz $WORKDIR
+
+# install opae
+RUN tar -zxf a10_gx_pac_ias_1_1_pv_rte_installer/components/a10_gx_pac_ias_1_1_pv.tar.gz sw
```

Comment:
Looks like there is missing unpacking step for a10_gx_pac_ias_1_1_pv_rte_installer.tar.gz

Reply:
ADD unpacks it.